client/src/Pages/Home.vue

```diff
@@ -25,6 +25,15 @@
                     <span aria-hidden="true">&times;</span>
                 </button>
             </div>
+            <div v-if="createEventProgress" class="alert alert-warning alert-dismissible">
+                <div class="d-inline mr-2">
+                    <img class="loading-logo" src="../assets/bluelogo.png" width="50" height="50">
```

Comment:
I think, the functionality that we have talked about did not implemented in this changes, it was taking loading in background to do not prevent any other functionality when create event is loading. We can discuss it.

Reply:
the loading screen that didn't allow you to use any other functionality has been removed in this pull request. I have added an alert that show you when haaukins is creatian an event.

Reply:
owh ok, then. Sorry I am not well at this syntax. :)